tests/phpunit/class-wc-stripe-api-sdk-test.php

```
@@ -0,0 +1,245 @@
+ * Tests for the Stripe SDK integration in WC_Stripe_API.
+ *
+ * @package WooCommerce\Tests
+ */
+class WC_Stripe_API_SDK_Test extends WP_UnitTestCase {
+
+	/**
+	 * Original SDK instance to restore after each test.
+	 *
+	 * @var \Stripe\StripeClient|null
+	 */
+	private $original_sdk = null;
```

PHPMD (2.15.0)

[warning] Avoid unused private fields such as '$original_sdk'. (undefined)

(UnusedPrivateField)